tensorflow/compiler/mlir/lite/tests/ops.mlir

```diff
@@ -678,6 +678,66 @@ func.func @testMaxPool2DWrongOperandStorageType(tensor<1x7x7x16x!quant.uniform<i
 
 // -----
 
+// CHECK-LABEL: testMaxPool3D
+func.func @testMaxPool3D(tensor<256x32x32x32x3xf32>) -> tensor<?xf32> {
+^bb0(%arg0: tensor<256x32x32x32x3xf32>):
```

Comment:
This is the "old" form, you can just roll the bb args into the function signature as you did in the other test.

Reply:
Done